data/test/scenarios/backstab.cfg

```diff
@@ -82,14 +82,80 @@
 #enddef
 
 # The test uses a Rogue to do the attack, which does 6x3 damage before the ability is applied
+
+#####
+# API(s) being tested: [damage]+[filter_opponent]formula
+##
+# Starting state:
+# Side 1 leader Alice (Elvish Archer)
+# Side 2 leader Bob (Orcish Grunt)
+##
+# Actions:
+# Bob is put on Void terrain (0% defense) and given 100 hp.
+# A Rogue is spawned north and south of Bob.
+# A Rogue attacks Bob.
+##
+# Expected end state:
+# Bob gets hit all three times and ends with 100-(6 * 3 * 2) hp.
+#####
 {TEST_BACKSTAB "backstab_simple" "n,s" (6 * 3 * 2) ()}
+
+#####
+# API(s) being tested: [damage]+[filter_opponent]formula
+##
+# Starting state:
+# Side 1 leader Alice (Elvish Archer)
+# Side 2 leader Bob (Orcish Grunt)
+##
+# Actions:
+# Bob is put on Void terrain (0% defense) and given 100 hp.
+# A Rogue is spawned north, southeast, and southwest of Bob.
```

Comment:
They're spawned south, northwest and northeast of Bob (on the hexes that Bob is `n,se,sw` of).